src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Cells/HealthCertificateQRCodeCell.swift

```diff
@@ -118,6 +118,7 @@ class HealthCertificateQRCodeCell: UITableViewCell, ReuseIdentifierProviding {
 		let validityStateStackView = UIStackView()
 		validityStateStackView.alignment = .center
 		validityStateStackView.axis = .horizontal
+		validityStateStackView.distribution = .fillProportionally
```

Comment:
`fillProportionally` in combination with a fixed-size image doesn't look quite right to me, did the default `fill` not work? If not, maybe some hugging/content compression priorities were not set correctly?

Reply:
`.fill` works fine too - I changed it